### PR TITLE
Changed package to materials, refactored Warehouse, Stocktaking, added enam Material

### PR DESCRIPTION
**Pull request by lyaleko**
Please review code.

### CODE REVIEW

**Review by vpetrushka — APPROVED** [review]

File: src/test/java/calculator/CalculatorTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    @Test
+    public void multiplyNumbersTest(){
+        Assert.assertEquals("Multiply is incorrect: ",27, this.calc.multiply(x,y));
+    }
+
+    @Test
+    public void divideNumbersTest(){
+        Assert.assertEquals("Divide is incorrect: ",3, this.calc.divide(x,y));
+    }
```
Comment (vpetrushka) [inline src/test/java/calculator/CalculatorTest.java:47]: missing one test to divide by zero

File: src/test/java/calculator/RectangleTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    public void figureSetLengthTest() {
+        var value = 25;
+        figure.setLength(value);
+        Assert.assertTrue("Rectangle is not correct", value== figure.getLength());
+    }
+
+    @Test
+    public void figureSetWidthTest() {
+        var value = 10;
+        figure.setWidth(value);
```
Comment (vpetrushka) [inline src/test/java/calculator/RectangleTest.java:56]: why not to set value straight away?

File: src/test/java/calculator/RectangleTest.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    @Test
+    public void figureGetWidthTest(){
+        Assert.assertTrue("Rectangle is not correct", width== figure.getWidth());
+    }
+
+    @Test
+    public void figureSetLengthTest() {
+        var value = 25;
+        figure.setLength(value);
```
Comment (vpetrushka) [inline src/test/java/calculator/RectangleTest.java:49]: why not to set value straight away?

File: src/test/java/calculator/RectangleTest.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    @Test
+    public void figureSetWidthTest() {
+        var value = 10;
+        figure.setWidth(value);
+        Assert.assertTrue("Rectangle is not correct", value==figure.getWidth());
+    }
+
+}
```
Comment (vpetrushka) [inline src/test/java/calculator/RectangleTest.java:60]: missing area test

File: src/test/java/calculator/CalculatorTest.java (hunk 1 above)
Comment (lyaleko) [reply src/test/java/calculator/CalculatorTest.java:52]: added test divide by zero

File: src/test/java/calculator/RectangleTest.java (hunk 2 above)
Comment (lyaleko) [reply src/test/java/calculator/RectangleTest.java:56]: this is just another way to set value

File: src/test/java/calculator/RectangleTest.java (hunk 3 above)
Comment (lyaleko) [reply src/test/java/calculator/RectangleTest.java:49]: this is just another way to set value

**Review by vpetrushka — APPROVED** [review]
